.gitignore

```diff
@@ -3,3 +3,4 @@ src/main/resources/database.properties
 .classpath
 .project
 .settings/
+.vscode
```

Comment:
This is not necessary. You should not change the gitignore file of the rest of the workers.

Reply:
Please, remove this line on other commit